main/src/main/java/org/mobilitydata/gtfsvalidator/validator/StopTimesGeographyIdPresenceValidator.java

```diff
@@ -63,4 +63,14 @@ public void validate(GtfsStopTime stopTime, NoticeContainer noticeContainer) {
     //              stopTime.locationId()));
     //    }
   }
+
+  @Override
+  public Boolean shouldCallValidate(ColumnInspector header) {
+    if (header.hasColumn("stop_id")
+        || header.hasColumn("location_group_id")
+        || header.hasColumn("location_id")) {
```

Comment:
Reference the columns by their constant value:
```suggestion
    if (header.hasColumn(GtfsStopTime.STOP_ID_FIELD_NAME)
        || header.hasColumn(GtfsStopTime.LOCATION_GROUP_ID_FIELD_NAME)
        || header.hasColumn(GtfsStopTime.LOCATION_ID_FIELD_NAME)) {
```